client/signup/step-wrapper/style.scss

```diff
@@ -92,3 +92,79 @@
 		margin-left: auto;
 	}
 }
+
+.step-wrapper__image {
+	margin-top: 24px;
+	display: none;
+
+	@include break-small {
+		display: block;
+	}
+	
+	img {
+		width: 50%;
+	}
+}
+
+.layout:not( .dops ) .step-wrapper {
+	max-width: 960px;
+	margin: 0 auto;
+
+	// Some steps (like plans) need a larger
+	// width column.
+	&.is-wide-layout {
+		max-width: 1040px;
+	}
+
+	// Some steps (intent gathering) use a horizontal layout
+	&.is-horizontal-layout {
+		align-items: flex-start;
+		margin-top: 20%;
+		padding: 0 20px;
+
+		@include break-small {
+			display: flex;
+		}
+
+		.step-wrapper__header {
+			display: block;
+			margin-top: 0;
+		}
+		.step-wrapper__content {
+			margin-left: auto;
+		}
```

Comment:
Some of these are my first implementation and I think it might be better now as followed
```css
&.is-horizontal-layout {
    align-items: flex-start;
    justify-content: space-between;
    max-width: 1024px;
    padding: 0 20px;
    margin-top: 25vh

    .step-wrapper__header {
        display: block;
	    margin-top: 0;
    }
}
```

Reply:
Nice, I applied this and it's looking even better 👍